### PR TITLE
Bumped the Mesos and mesos-modules packages.

### PR DESCRIPTION
**Pull request by greggomann**
## High Level Description

This PR updates the `Mesos` and `mesos-modules` packages.

## Related Issues

  - [CORE-1067](https://jira.mesosphere.com/browse/CORE-1067) Filter results of `/master/slaves` and the v1 call `GET_AGENTS`
  - [DCOS-17194](https://jira.mesosphere.com/browse/DCOS-17194) Rename .cni config files to .conf

## Checklist for all PR's

  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [x] Change log from the last version integrated (this should be a link to commits for easy verification and review): [Mesos diff](https://github.com/mesosphere/mesos/compare/6275099d4d5e0eadc5d39cf7a28d6d7930788680...376cf43788fe84a9982dd09b05f5207a9caf4cf6), [mesos-modules diff](https://github.com/dcos/dcos-mesos-modules/compare/2a5a02f1de47dcbe44844f7358adf85ea630cf4a...54020bfcc858ae0491c49013ebcdf46269e1d173)
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by greggomann** [comment]
@mesosphere-mergebot bump-ee

**Comment by greggomann** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by orsenthil** [comment]
@greggomann  - the upgrade-onprem test is not successful yet. This needs to turn green before it can be merged.

**Comment by jgehrcke** [comment]
Changing to 'work in progress' after discussion with @sambatyon

File: packages/mesos/buildinfo.json [hunk 1]
```diff
@@ -3,8 +3,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "6275099d4d5e0eadc5d39cf7a28d6d7930788680",
-    "ref_origin" : "dcos-mesos-master-013f7e21"
+    "ref": "376cf43788fe84a9982dd09b05f5207a9caf4cf6",
+    "ref_origin" : "dcos-mesos-master-e068e84c"
```
Comment (adam-mesos) [inline packages/mesos/buildinfo.json]: Not as recent as `dcos-mesos-master-1b15ff7a` in https://github.com/dcos/dcos/pull/1753
Might just drop this in favor of #1753

**Comment by sambatyon** [comment]
Work is being continued in #1753, please someone with rights could close the PR?

**Pull request closed without merging**